sunscreen_docs/src/advanced/batching/dot_product.md

```diff
@@ -250,7 +250,7 @@ fn main() {
 
     // 3. Make our runtime, generate keys, encrypt our data, run our
     // program, and check the result
-    let runtime = Runtime::new(&fhe_program.metadata.params).unwrap();
+    let runtime = Runtime::new_fhe(&fhe_program.metadata.params).unwrap();
```

Comment:
Got through each md file and make sure the reading is still consistent with the examples.

Reply:
As discussed in our sync, we'll do this after checking in.